src/components/Create/components/pages/hooks/FormDispatchWatch.ts

```diff
@@ -2,6 +2,7 @@ import { ActionCreatorWithPayload } from '@reduxjs/toolkit'
 import { FormInstance } from 'antd'
 import { useWatch } from 'antd/lib/form/Form'
 import { useAppDispatch } from 'hooks/AppDispatch'
+import { isEqual } from 'lodash'
```

Comment:
not 100% sure if this makes a difference to the bundle in our case, but to be safe I tend to:

```suggestion
import isEqual from 'lodash/isEqual'
```